Backend/dev/migrations/dynamic-offer-driver-app/0320-cancellationDues-column-in-fareparams.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE atlas_driver_offer_bpp.fare_parameters ADD column customer_cancellation_dues int;
+
+ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD column cancellation_fee int DEFAULT 10 NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD column driver_pickup_distance_bound bigint DEFAULT 500 NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD column driver_booking_time int DEFAULT 600 NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD column dispute_limit int DEFAULT 3 NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD column num_of_cancellations_allowed int DEFAULT 3 NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD column driver_distance_to_pickup_bound int DEFAULT 50 NOT NULL;
```

Comment:
do not correspond to Haskell code:    
```
  cancellationFee :: B.C f Money,
  driverDistanceTravelledOnPickupThreshold :: B.C f Meters,
  driverTimeSpentOnPickupThreshold :: B.C f Int,
  cancellationFeeDisputeLimit :: B.C f Int,
  driverDistanceToPickupThreshold :: B.C f Meters,
  numOfCancellationsAllowed :: B.C f Int,
```

Reply:
Oh just now changed the names. Correcting it